web/wp-content/plugins/ad-stack/plugin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/*
+Plugin Name: TMBI Ad Stack
+Version 1.0.0
+Description: Show ads on TMBI sites
+Author: PLT Team
+Text Domain: tmbi-ad-stack
+*/
+
+include( 'inc/ads-global-targeting.php' );
+
+class AdStack {
+	public function __construct() {
+		add_action( 'wp_enqueue_scripts', array($this, 'register_scripts') );
```

Comment:
When adding actions and filters, we should not break `remove_action` or `remove_filter` APIs. Anonymous class instances can't be removed easily (without looping through the global `$wp_filter` and checking ìnstanceof` and the function name).

If classes are necessary, static functions won't break `remove_action`. In this case, however, classes are only used for namespacing, is that right?